### PR TITLE
RDAPI-29587 Manually create secrets

### PR DESCRIPTION
**Pull request by vobrien-axway**
Add instructions to disable automatic secret creation from the values file and instead manually create them in kubernetes

## Deploy preview link

http://pr-2665.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/operational_insights/production_setup/op_insights_setup_prod_helm/

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

File: content/en/docs/operational_insights/production_setup/op_insights_setup_prod_helm.md [hunk 1]
```diff
@@ -282,4 +282,73 @@ The following example shows how you can create a secret that keeps the API Manag
...
 
-As a result of this section, you have set up your `myvalues.yaml` Helm chart to run Operational Insights in a single instance in a production environment.
\ No newline at end of file
+As a result of this section, you have set up your `myvalues.yaml` Helm chart to run Operational Insights in a single instance in a production environment.
+
+## Use a secret for other credentials
+
+Each component uses secrets to store credentials such as username and password. When deployed, the chart will take the value entered in the field in your `values.yaml` file and generate a secret from that value. If you would prefer not to have the credentials entered in your values file, you can disable the secrets for that component and manually create the secret. Then when the helm chart is deployed, these secrets will be picked up. Examples of how to do this for each component are below.
+
+### APM Server
```
Comment (andreamussap) [inline content/en/docs/operational_insights/production_setup/op_insights_setup_prod_helm.md:291]: What's APM? please expand this

**Comment by kmercierwhite** [comment]
@vobrien-axway, FYI I think we want to merge this into master

File: content/en/docs/operational_insights/production_setup/op_insights_setup_prod_helm.md (hunk 1 above)
Comment (vobrien-axway) [reply content/en/docs/operational_insights/production_setup/op_insights_setup_prod_helm.md:291]: It is a component of AAOI just like all the others. I believe it stands for Application Performance Monitoring, but throughout all ELK stack documentation it is referred to as APM Server

File: content/en/docs/operational_insights/production_setup/op_insights_setup_prod_helm.md [hunk 2]
```diff
@@ -282,4 +282,73 @@ The following example shows how you can create a secret that keeps the API Manag
 
    {{< alert title="Note" >}}Refer to `repository.axway.com` for the most recent version of the Helm chart. {{< /alert >}}
 
-As a result of this section, you have set up your `myvalues.yaml` Helm chart to run Operational Insights in a single instance in a production environment.
\ No newline at end of file
+As a result of this section, you have set up your `myvalues.yaml` Helm chart to run Operational Insights in a single instance in a production environment.
+
+## Use a secret for other credentials
+
+Each component uses secrets to store credentials such as username and password. When deployed, the chart will take the value entered in the field in your `values.yaml` file and generate a secret from that value. If you would prefer not to have the credentials entered in your values file, you can disable the secrets for that component and manually create the secret. Then when the helm chart is deployed, these secrets will be picked up. Examples of how to do this for each component are below.
```
[the remark this answers is not in the document]
Comment (vobrien-axway) [reply content/en/docs/operational_insights/production_setup/op_insights_setup_prod_helm.md]: changes made. The first refers to, not a singular field, but a field that is available in each of the different components, and can be configured the same or differently in each case. I have reworded it in the hopes it causes less confusion

**Pull request closed without merging**